Makefile

```diff
@@ -1,5 +1,11 @@
 VERSION := v0.1.0
 
+ORG_PATH := github.com/Azure
+PROJECT_NAME := eraser
+REPO_PATH := $(ORG_PATH)/$(PROJECT_NAME)
+BUILD_COMMIT := $(shell git rev-parse --short HEAD)
```

Comment:
If we include commit we probably also need if the tree is dirty.

```
test -n "$(git status --porcelain)"
```

As a note, go1.18 compiled binaries already includes this info.